src/openms/include/OpenMS/ANALYSIS/OPENSWATH/PeakIntegrator.h

```diff
@@ -517,6 +517,44 @@ namespace OpenMS
 
     void getDefaultParameters(Param& params);
 
+    /**
+      @brief Fit the given peak (a MSChromatogram) to the EMG peak model
+
+      It is desirable to use this method when dealing with saturated or cutoff peaks.
+      The output is a reconstruction of the input peak. Additional points are added
+      to produce a peak with similar intensities on boundaries' points.
+
+      Inspired by the results of the paper:
+      Yuri Kalambet, Yuri Kozmin, Ksenia Mikhailova, Igor Nagaev, Pavel Tikhonov
+      Reconstruction of chromatographic peaks using the exponentially modified Gaussian function
+
```

Comment:
We should add a note that all optimal gradient descent parameters are currently hard coded to allow for a simplified user interface.  If this feature proves useful and robust during routine use, later iterations can expand the EMG fitting methods to its own class to allow for additional gradient descent techniques as well as parameterization.